src/scripts/Blacklist.elm

```diff
@@ -0,0 +1,12 @@
+module Blacklist exposing (contains, length)
+
+
+length : Int
+length =
+    2
+
+
+contains : { a | user : String, name : String } -> Bool
```

Comment:
Why are those packages blacklisted? Renamed use/project names?

Reply:
`tea-component` was manually deleted ([source](https://groups.google.com/d/msg/elm-dev/iqErmKHnLDY/DXPOSOwsAQAJ)). I don't know the story behind `elm-i18next`, but if you check Elm Package, there are no published versions. I'm guessing it was manually deleted as well.